snapshotter/README.md

```diff
@@ -29,12 +29,12 @@ A basic configuration would look like:
 ```
 # tell containerd to use this particular snapshotter
 [plugins."io.containerd.grpc.v1.cri".containerd]
-  snapshotter = "cvmfs-snapshotter"
```

Comment:
Here it is important to keep the "cvmfs" prefix to differentiate from other snapshotters